code/modules/mob/living/simple_animal/hostile/hostile.dm

```diff
@@ -18,8 +18,20 @@
 	hunger_enabled = 0//Until automated eating mechanics are enabled, disable hunger for hostile mobs
 	var/shuttletarget = null
 	var/enroute = 0
+
+	// Vars to help find targets
 	var/list/targets = list()
 	var/attacked_times = 0
+	var/list/target_type_validator_map = list()
+	var/lowest_health = INFINITY // Max you can get
+	var/atom/T = null

```

Comment:
Nit pick vol two. This is unnecessary. Expand the callback signature to include the current picked target, since it might be useful information. `validator_living(var/mob/living/L, var/atom/current)`.

I wonder if the same is worth doing for lowest health? Actually, that can be accessed if you cast current to mob and fetch it there!